tools/ds-identify

```diff
@@ -753,6 +753,12 @@ dscheck_CloudSigma() {
     return $DS_NOT_FOUND
 }
 
+dscheck_Akamai() {
```

Comment:
Could we please add unit-test coverage of this in `tests/unittests/test_ds_identify.py`

https://canonical-cloud-init.readthedocs-hosted.com/en/latest/development/datasource_creation.html#add-tests-for-ds-identify